openhtf/util/measurements.py

```diff
@@ -119,20 +117,22 @@ class Measurement(  # pylint: disable=no-init
   """
 
   def Doc(self, docstring):
+    """Set this Measurement's docstring, returns self for chaining."""
     self.docstring = docstring
     return self
 
   def WithUnits(self, units):
-    """Declare the units for this Measurement."""
+    """Declare the units for this Measurement, returns self for chaining."""
     self.units = units
     return self
 
   def WithDimensions(self, *dimensions):
-    """Declare dimensions for this Measurement."""
+    """Declare dimensions for this Measurement, returns self for chaining."""
```

Comment:
meh don't think it really matters.  i only made the one just say 'chainable' because it wouldn't fit on one line otherwise, and it was already clear what 'chainable' meant because of these other comments.